core/capabilities/triggers/logevent/trigger.go

```diff
@@ -84,6 +84,10 @@ func newLogEventTrigger(ctx context.Context,
 	callbackCh := make(chan capabilities.TriggerResponse, defaultSendChannelBufferSize)
 	ticker := time.NewTicker(time.Duration(logEventConfig.PollPeriod) * time.Millisecond)
 
+	if logEventConfig.QueryCount == 0 {
+		logEventConfig.QueryCount = 20
```

Comment:
nit: make the default a const

Reply:
Sure, I'll make this change in a future PR